deployment/gcp/deploy.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "applicationType": "service",
+    "dockerFilePath": "extension/",
+    "name": "Extension",
+    "properties": {}
```

Comment:
Remove this empty properties.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1005/commits/8bd218908ba6fae9ac50347683860496daed662e